src/node_api_helpers.h

```diff
@@ -132,23 +132,13 @@ namespace Napi {
   // like Nan, always using the current one?
   class Callback {
    public:
-    Callback() {
-      napi_env env;
-      napi_get_current_env(&env);
-      HandleScope scope(env);
-      napi_value obj;
-      napi_create_object(env, &obj);
-      napi_create_reference(env, obj, 1, &handle);
-    }
+    Callback() {}
```

Comment:
Yes, `Callback() : handle{nullptr} {}`. Otherwise the pointer is uninitialized and although it might not be a problem right now, any small change which relies on the pointer being initialized would implode.